### PR TITLE
Multipart mime 3487 v3

### PR DESCRIPTION
**Pull request by catenacyber**
## Ticket

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/3487

See https://github.com/OISF/suricata/pull/9424

#1371 with fixes to pcap 
@inashivb do you approve ?

### CODE REVIEW

File: tests/mime/mime-dec-parse-full-msg-test01/test.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# *** Add configuration here ***
+
```
Comment (jufajardini) [inline tests/mime/mime-dec-parse-full-msg-test01/test.yaml:2]: We usually ask folks to remove this. :P

File: tests/mime/mime-dec-parse-long-filename01/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Test Description
+
+Test some mimre processing
```
Comment (jufajardini) [inline tests/mime/mime-dec-parse-long-filename01/README.md:3]: Is mimre a real term, or is it a typo? Couldn't find it on google...

Comment (catenacyber) [reply tests/mime/mime-dec-parse-long-filename01/README.md:3]: Oops, changing

File: tests/mime/mime-dec-parse-full-msg-test01/test.yaml (hunk 1 above)
Comment (catenacyber) [reply tests/mime/mime-dec-parse-full-msg-test01/test.yaml:2]: Ok.

Why is it not removed automatically ?

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata-verify/pull/1383

**Pull request closed without merging**